onnx/defs/nn/defs.cc

```diff
@@ -2172,4 +2172,61 @@ ONNX_OPERATOR_SET_SCHEMA(
              {{"Processed_STD"}, "Add", {"STD", "Epsilon"}},
              {{"Y"}, "Div", {"X_variance", "Processed_STD"}}})));
 
+static const char* StringConcat_ver11_doc = R"DOC(
+Concatenates strings in the input tensor to form a larger string. 
+A separator can be specified to separate entries in the resulting string. 
+One single space is used as the default.
+The input tensor can be of rank 1 or 2.
+The resulting tensor will be of rank 0 or 1.
```

Comment:
Could you elaborate more on the rank of input and output tensors? in the example, input is rank 2, and output is rank 2, too.

Reply:
Thank you. I will revise the description and push again